File: HOME/Dz - 2/main2.py
#-------------Hw-2-#2----------------------------#
x = int(input("Введите число: "))
y = int(input("Введите число: "))
z = int(input("Введите число: "))
m = [x,y,z]
numbers = int(input("Выберите: \n1 - максимум из трёх \n2 - минимум из трёх \n3 - среднеарифметическое из трёх"))
if numbers == 1:
    b = max(m)
    print(b)
elif numbers == 2:
    b = min(m)
    print(b)
elif numbers == 3:
    if y < x < z or z < x < y:
        print(x)
    elif x < y < z or z < y < x:
        print(y)
    else:
        print(z)
else:
    print("Вы не выбрали ни один из вариантов")


# if numbers == 1:
#     m = x
#     if m < y:
#         m = y
#     if m < z:
#         m = z
#     print(m)
# elif numbers == 2:
#     m = [x, y, z]
#     b = min(m)
#     print(b)
#
#g = (x+y+z)/3

File: HOME/Dz - 2/main2.js
// Ошибка в нахождении по пути 8 - 21
x = +prompt("Введите число: ")
y = +prompt("Введите число: ")
z = +prompt("Введите число: ")
m = [x,y,z]

numbers = +prompt("Выберите: \n1 - максимум из трёх \n2 - минимум из трёх \n3 - среднеарифметическое из трёх")
if (numbers == 1) {
    b = max(m)
    console.log(b)
} else if (numbers == 2) {
    b = min(m)
    console.log(b)
} else if (numbers == 3) {
    if (y < x < z || z < x < y) {
        console.log(x)
    } else if (x < y < z || z < y < x) {
        console.log(y)
    } else {
        console.log(z)
    }
} else {
    console.log("Вы не выбрали ни один из вариантов")
}